.github/workflows/test-e2e-runtime.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     continue-on-error: true
     strategy:
       matrix:
-        runtime: ["node", "go", "python", "quarkus", "springboot", "typescript", "rust"]
```

Comment:
Why has been python removed here?

Reply:
Because the Pack builder used in the E2Es is not supported in the new middleware.  I suspect we'll have back-portable support added in the next few days.